### PR TITLE
#342/UI/Desktop/UserDogs

### LINKED ISSUE #342: UI |  Desktop | User Dogs
1 - 1 zmiana pozycji przycisku

  Desktop /User Dogs/5

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
closes #342 

800px
image

1024 && desktop:
image

### CODE REVIEW

**Review by n0macx — COMMENTED** [review]
There are many views with just buttons like this one. Maybe it would be worth it, not to put max-width with classes on many elements but use ColumnWrapper - with added max-width property and alignment defined by props. It would be reusable and we could use it on the rest of screens ?

File: client/src/PagesBody/UserDogsPage/UserDogPage.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const UserDogPage = () => {
   }, []);
 
   return (
-    <ColumnWrapper paddingLeftRight={1} paddingTop={0.5}>
+    <ColumnWrapper paddingLeftRight={1} paddingTop={0.5} className="user-dogs-column-wrapper">
```
Comment (vieraboschkova) [inline client/src/PagesBody/UserDogsPage/UserDogPage.js:38]: Thats a long classname :)

Comment (KatarzynaChmielecka) [reply client/src/PagesBody/UserDogsPage/UserDogPage.js:38]: yes, but 'user-dogs' already exist :)

**Review by vieraboschkova — APPROVED** [review]
Good job!

**Comment by KatarzynaChmielecka** [comment]
> There are many views with just buttons like this one. Maybe it would be worth it, not to put max-width with classes on many elements but use ColumnWrapper - with added max-width property and alignment defined by props. It would be reusable and we could use it on the rest of screens ?

we can make a refactor in the further stage of the project :)

**Review by vieraboschkova — APPROVED** [review]

**Pull request merged**